pkg/synthesis/k8sPolicies.go

```diff
@@ -231,7 +231,9 @@ func createSelector(con symbolicexpr.Conjunction) policySelector {
 	for _, a := range con {
 		switch {
 		case a.IsTautology():
+			res.cidrs = a.GetBlock().ToCidrList()
 		case a.IsAllGroups():
+			// leaving it empty - will match all labels
```

Comment:
```suggestion
			// leaving it empty - will match all labels
			// todo: should be fixed when supporting namespaces 
```

Reply:
the policy has a namespace, and an empty labels selector is scoped to that namespace only.. so it is needed to be extended also with namespace selector, to apply for all namespaces as well.